ert3/storage/_storage.py

```diff
@@ -406,6 +424,7 @@ def add_ensemble_record(
     workspace: Path,
     record_name: str,
     ensemble_record: ert3.data.EnsembleRecord,
+    ensemble_size: Optional[int] = None,
```

Comment:
Is it necessary to pass `ensemble_size`? It is part of the ensemble record.

Reply:
When this function is called from `_record.py::load_record()` there is no information about ensemble size yet. So the blob is added to a Record one. Then when we later 'prepare' the experiment we know the ensemble size and use this in `_storage.py::_response2records()` to create a Record with this many copies.

Reply:
Yes, I misunderstood how it works, clear now.